app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -888,6 +887,23 @@ impl Node {
                 <+ visualization_visible.not().and(&no_error_set);
 
 
+            // === Bounding Box ===
+
+            let visualization_size             = &model.visualization.frp.size;
+            visualization_enabled_and_visible <- visualization_enabled && visualization_visible;
+            bounding_box_input                <- all4(
+                &new_size,&position,&visualization_enabled_and_visible,visualization_size);
+            out.source.bounding_box <+ bounding_box_input.map(
+                |(node_size,node_position,visualization_enabled_and_visible,visualization_size)| {
```

Comment:
You don't need `{}` for single expressions.

Reply:
Uhhh, thanks, it slipped my attention after several refactorings of the code here made it unnecessary 😕